packages/fxa-settings/src/lib/nimbus/index.ts

```diff
@@ -31,27 +32,33 @@ export interface NimbusResult {
  */
 export async function initializeNimbus(
   clientId: string,
+  previewEnabled: boolean = false,
   context: NimbusContextT
 ): Promise<any> {
   const MAX_TIMEOUT_MS = 1000;
   const body = JSON.stringify({
     client_id: clientId,
     context,
   });
+  const nimbusPreview =
+    searchParams(window.location.search).nimbusPreview || previewEnabled;
```

Comment:
nit: This might cause some confusion since it overrides the argument value.  if someone is only looking at the types, they wouldn't know; they have to look at the function definition.

Reply:
@chenba Oh fair point. Do you think switching them around would suffice, or should we just stick with one way to do this (env)?

The downsides of only using one way, is that it makes it harder for two different problems:
- env only: makes it difficult for devs to change the value in stage/prod for testing.
- query param only: makes it flaky to know if you were always in an experiment as navigation happens (and potentially params are dropped).

My intuition is saying that switching them would fix the confusion and retain the flexibility, but I'm also fine with making changes.

Reply:
I've gone and pushed my recommendation for the sake of having this landed before the next release, but if you feel strongly, please let me know and I can put a follow-up patch.

Leaving unresolved till then.

Reply:
Is moving the fallback logic to the call site of initializeNimbus possible?

Reply:
That's a great idea - I put this in a follow-up PR here if you'd like to review it: https://github.com/mozilla/fxa/pull/18622